backend/schema/corpora_schema.md

```diff
@@ -0,0 +1,323 @@
+    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will be summed during
+    curation.
+*   **Type annotations or inferrable types**. For both expression values and metadata, we don't want to have to guess at the type. Typing is already explicit
+    in anndata, loom, Seurat and Bioconductor formats.
+*   **No PII**. No metadata can be personally identifiable: no names, dates of birth, specific locations, etc. There’s a
+    [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing).
+
+
+#### Matrix Layers
+
+The count matrix itself can exist in several forms. Generally, there is a “raw” count matrix without scaling, filtering, normalization, etc. And there is
+a “final” matrix that is used in the publication with QC and various corrections. There can also be several intermediate matrices.
+
+These different transformations of the count matrix are called “layers” in several file formats. Submissions to Corpora must contain at least one layer,
+the “raw” layer. They may optionally also have a “final” layer used for presentation, and each layer must be identified.
+
+
+### Remix Metadata
+
+The remaining part of the schema defines what’s needed for the Corpora remix. The words “must” and “requirement” are used throughout, but note the
+discussion above regarding curation policies as well the [caution against](#Software-vs-policy) introducing any
+software dependencies on these fields.
+
+
+#### Integration Metadata
+
+To support data integration, each cell must have the following metadata:
+
+**Field name**|**Constraints**
+:--|:--
+tissue|string
+assay|string
+disease|string
+cell\_type|string
+sex|“male”, “female”, “unknown”, or “na”
+ethnicity|string, “na” if non-human
+development\_stage|string
+
+In addition to these free text fields (except sex), each cell must also have ontology values:
+
+**Field name**|**Constraints**
+:--|:--
+tissue\_ontology|UBERON term
+assay\_ontology|EFO term
+disease\_ontology|MONDO term or [PATO:0000461](http://bioportal.bioontology.org/ontologies/PATO?p=classes&conceptid=PATO%3A0000461)
+cell\_type\_ontology|CL term
+ethnicity\_ontology|HANCESTRO term, “na” if non-human
+development\_stage\_ontology|HsapDv term if human, child of EFO:0000399 otherwise
+
+
+The ontology fields are all “best effort” fields. We expect that there will be times that no ontology term exists, especially for cell_type. In those cases,
+the ontology field may be left empty. Also, we don’t require submitters to include these fields in their submissions. If the submitter does not include
+ontology fields, then as part of the curation process, Corpora will insert appropriate ontology values based on the text from the submitters. If the
+submitter does include ontology values, then Corpora just uses those. Finally, tissue and tissue_ontology can be appended with “(cell culture)” or
+“(organoid)” if appropriate.
```

Comment:
I'd move the sentence about appended with the parenthesis right underneath the tables to make it more clear. Right now this important nugget is a bit hidden. Or even mention it directly in the contraints.